pkg/services/live/pipeline/config.go

```diff
@@ -93,6 +133,23 @@ type ChannelRules struct {
 	Rules []ChannelRule `json:"rules"`
 }
 
+func checkRulesValid(orgID int64, rules []ChannelRule) (ok bool, reason string) {
+	t := tree.New()
+	defer func() {
+		if r := recover(); r != nil {
```

Comment:
Instead of handling panic we better modify router tree further and return errors on inserting child, but let's do this a bit later.